frontend/src/lib/apiClient.js

```diff
@@ -75,6 +75,18 @@ const getRecipes = (search) => {
 
 const getRecipe = id => apiClient.get(`api/v1/recipes/${id}`);
 
+const createRecipe = recipe => apiClient.post('api/v1/recipes#create', {
```

Comment:
The `create` part shouldn't be in the URL. `api/v1/recipes` points to the `RecipesController`, and `create` in Rails is just the method that gets called when you `POST`.